e2e/README.md

```diff
@@ -0,0 +1,51 @@
+# Cypress
+
+## Local setup
+On linux do not trust it to be effortless, who knows on mac?
+
+1. go to [Chromium Downloads Tool](https://chromium.cypress.io/) and download the appropriate version
+
+    1. extract content to `./cypress/browsers`
+
+2. install node dependencies
+
+    ```
+    yarn install
+    ```
+
+3. configurations file
+   1. a `./config/example.json` is available as a basis to create `./config/local.json`
+   1. change the values where appropriate
+
+
+## Executing tests
+1. to open cypress
+    ``` bash
+    # using /config/local.json
+    yarn run cy:open --browser ~/repos/dfe/manage-courses-frontend/cypress/browsers/chrome-linux/chrome
+    ```
+
+    ``` bash
+    # native
+    yarn run cy:open --env 'email=someone@test.com,password=change me' --browser ~/repos/dfe/manage-courses-frontend/cypress/browsers/chrome-linux/chrome
```

Comment:
Should these values be environment variables / Settings?

Reply:
Yes they can be as well